docker/Dockerfile

```diff
@@ -149,10 +150,12 @@ WORKDIR /app
 COPY docker/conf /conf/
 COPY docker/bin/* /usr/local/bin/
 
-COPY --chown=hope:unicef . /app/
+COPY --chown=hope:unicef --from=production /app/src /app/src
```

Comment:
why doing this change and the below?

Reply:
@domdinicola the dist image by pip only copy .py files in site-packages. If I rm all src/* files and then run the collectstatics in entrypoint, the font files wont be available, since they will have already been deleted.

Reply:
we have to change the dist to include the static files in the package too